src/components/EventsModal/EditEventModal.jsx

```diff
@@ -0,0 +1,346 @@
+    });
+  };
+
+
+  const manipulateDate = () => {
+    const dateObject = new Date(Date.parse(event['date']));
+    var year = dateObject.getFullYear();
+    var month = dateObject.getMonth() + 1;
+    var day = dateObject.getDate();
+    var formattedDate = year + '-' + (month < 10 ? '0' : '') + month + '-' + (day < 10 ? '0' : '') + day;
+    setEventDate(formattedDate);
+  }
+
+  const manipulateStartTime = () => {
+    if (!eventData || !eventData.start_time) {
+      return '';
+    }
+    const time = eventData.start_time.substring(0, 5);
+    const hour = parseInt(time.substring(0, 2));
+    const minute = time.substring(3);
+    
+    // Format hour and minute with leading zeros
+    const formattedHour = (hour < 10 ? '0' : '') + hour;
+    const formattedMinute = (minute < 10 ? '0' : '') + minute;
+    console.log(formattedHour + ':' + formattedMinute);
+    setEventStartTime(formattedHour + ':' + formattedMinute);
+  };
+
+  const manipulateEndTime = () => {
+    if (!eventData || !eventData.end_time) {
+      return '';
+    }
+    const time = eventData.end_time.substring(0, 5);
+    const hour = parseInt(time.substring(0, 2));
+    const minute = time.substring(3);
+    
+    // Format hour and minute with leading zeros
+    const formattedHour = (hour < 10 ? '0' : '') + hour;
+    const formattedMinute = (minute < 10 ? '0' : '') + minute;
+    console.log(formattedHour + ':' + formattedMinute);
+    setEventEndTime(formattedHour + ':' + formattedMinute);
+  };
+
+  useEffect(() => {
+  //  setEventDate(date);
+  //  setEventStartTime(startTime);
+  //  setEventEndTime(endTime);
+   setEventData(event);
+   manipulateDate();
+  }, [isOpen]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'event' and 'manipulateDate'. Either include them or remove the dependency array. If 'setEventData' needs the current value of 'event', you can also switch to useReducer instead of useState and read 'event' in the reducer.